tf2onnx/rewriter/eye_matrix_generator_rewriter.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""
+tf2onnx.rewriter.eye_matrix_generator - supports tf.eye
+"""
+
+import numpy as np
+
+from tf2onnx import utils
+from tf2onnx.graph_matcher import OpTypePattern, GraphMatcher
+
+# pylint: disable=invalid-name,unused-argument,missing-docstring
+
+
+def rewrite_eye(g, ops):
+    # tf.eye is implemented by a sub_graph which contained an op "MatrixDiag" un-supported directly in onnx
+    # but onnx op EyeLike can be used instead of the sub_graph
+    # for now, only tf.eye(tf.shape(x)[i]) are supported
+    # otherwise a generator op should be needed and onnx is going to support it so wait for now.
+    # the following pattern only match sub_graph of tf.eye(tf.shape(x)[i]), not tf.eye(tf.shape(x)[0], tf.shape[1])
+    pattern = \
```

Comment:
tf.eye(tf.shape(x)[i]) => tf.eye(num_rows) ?
tf.eye(tf.shape(x)[0], tf.shape[1]) => tf.eye(num_rows, num_columns) ?

Reply:
yes, the following info is comment extracted from this tf func:

> num_columns: Optional non-negative `int32` scalar `Tensor` giving the number of columns in each batch matrix.  Defaults to `num_rows`.